gix/src/pathspec.rs

```diff
@@ -92,6 +93,8 @@ impl<'repo> Pathspec<'repo> {
         alias = "matches_path",
         alias = "git2"
     )]
+    #[momo]
+    #[allow(clippy::needless_lifetimes)]
```

Comment:
Probably this is a similar effect as observed with the `unused_mut` lint.

Reply:
Resolved by the new internals https://github.com/Byron/gitoxide/pull/992#issuecomment-1697285009